spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
+require 'emerald/parser'
+require 'emerald/compiler'
+
```

Comment:
I also thinking that DRY has little place in tests tbh.

There are lots of other opinions here. The thing that matters more is consistency. :)